packages/tooltip/src/tooltip.ts

```diff
@@ -61,3 +107,9 @@ export class Tooltip extends LitElement {
         `;
     }
 }
+
+declare global {
```

Comment:
TS practices question: when working with an event that can be emitted in multiple places but has a centralized type, should this interface be declared in multiple places or just in `@spectrum-web-components/overlay` where the `OverlayDisplayQueryDetail` declaration is?

Reply:
Good question!

I don't think there's any specific 'best practice' around this. I have generally leant towards doing this declaration in the file/module that emits the event. This seems most logical since then as long as you have imported that module the declaration will be imported. Having said that as you point out if it can be emitted in multiple places, it might make sense to just move it out to its own file so it can be imported appropriately?

I'm not sure if this is best or not though.

Reply:
Looks like this is a duplicate definition. I had already moved this one to [here](https://github.com/adobe/spectrum-web-components/blob/ad1554d3c5648f5b86efc6d7e2d3c0e0cf57897a/packages/overlay/src/overlay-types.ts#L41-L45)